website/docs/docs/collaborate/git/connect-github.md

```diff
@@ -12,45 +12,65 @@ Connecting your GitHub account to dbt Cloud provides convenience and another lay
 
 If you are the GitHub organization owner tasked with the installation of the dbt Cloud app in step 1, you will also be automatically personally authenticated after completion, so step 2 will be taken care of. This means teams of one only need to complete step 1.
 
-:::info Use GitHub On-Premise?
+**GitHub On-Premise**
+
 If you're using an On-Premises GitHub deployment, this method will not work for your account. Please instead reference our docs on [importing a project by git URL](/docs/collaborate/git/import-a-project-by-git-url) to set up your connection. This alternative connection method does not include the benefits of the native integration mentioned above.
-:::
+
+
+## Prerequisites
+
+- You _must_ be a **GitHub org admin** in order to [install the dbt Cloud application](/docs/collaborate/git/connect-github#installing-dbt-cloud-in-your-github-account). 
+- The GitHub org admin _must_ log into dbt Cloud with _Owner_ or _Account Admin_ [permissions](/docs/collaborate/manage-access/self-service-permissions) to integrate a GitHub environment that uses Organizations.
+- You may need to provide an extra dbt Cloud user account for your GitHub org admin until they complete the installation.
 
 ## Installing dbt Cloud in your GitHub account
 
-A GitHub organization owner needs to connect and configure the dbt Cloud app for their GitHub organization. If you are a team of one or wish to connect your personal GitHub account, then these instructions also apply to you.
+To connect your dbt Cloud account to your GitHub account: 
+
+1. Navigate to **Your Profile** settings by clicking the gear icon in the top right. 
 
-To link your dbt Cloud account to your team's GitHub account, navigate to Your Profile settings by clicking the gear icon in the top right. Select **Linked Accounts** from the left menu.
+2. Select **Linked Accounts** from the left menu.
 
 <Lightbox src="/img/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-github/github-connect.gif" title="Navigated to Linked Accounts under your profile"/>
 
-In the Linked Accounts section, you can set up your GitHub account connection to dbt Cloud by clicking **Link** to the right of GitHub. This redirects you to your account on GitHub where you will be asked to install and configure the dbt Cloud application. Select the GitHub organization and repositories dbt Cloud should access.
+3. In the **Linked Accounts** section, set up your GitHub account connection to dbt Cloud by clicking **Link** to the right of GitHub. This redirects you to your account on GitHub where you will be asked to install and configure the dbt Cloud application. 
+
+4. Select the GitHub organization and repositories dbt Cloud should access.
 
 <Lightbox src="/img/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-github/github-app-install.png" title="Installing the dbt Cloud application into a GitHub organization"/>
```

Comment:
This can be aligned with the previous step for clarity
```suggestion
   <Lightbox src="/img/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-github/github-app-install.png" title="Installing the dbt Cloud application into a GitHub organization"/>
```